editor/editor_tests/user_stats_test.cpp

```diff
@@ -9,7 +9,8 @@ namespace editor
 namespace
 {
 auto constexpr kEditorTestDir = "editor-tests";
-auto constexpr kUserName = "Vladimir BI";
+// This user has made only 9 changes and then renamed himself, so there will be no further edits from him.
+auto constexpr kUserName = "Nikita Bokiy";
```

Comment:
It's simpler to find an existing user, then to register in OSM and make any edits.
Also, it's highly unlikely that there will be another Nikita with this name.